nvme.c

```diff
@@ -2318,8 +2318,8 @@ static int fw_download(int argc, char **argv, struct command *cmd, struct plugin
 	}
 
 	buf = fw_buf;
-	if (cfg.xfer == 0 || cfg.xfer % 4096)
-		cfg.xfer = 4096;
+	if (cfg.xfer == 0 || cfg.xfer % 512 || cfg.xfer > 512)
```

Comment:
Does it mean for whatever cfg.xfer's value, we'll assign it to 512?

Reply:
Yes that's what it does. I wanted to change it to `cfg.xfer=512` but decided to keep it in case we support other size later.

Reply:
Updated with `cfg.xfer=512`.